src/walletConnect/screens/useIsDappListed.ts

```diff
@@ -1,10 +1,14 @@
 import { useSelector } from 'react-redux'
 import { dappsListSelector } from 'src/dapps/selectors'
+import { WalletConnectDisplayedInfo } from 'src/walletConnect/v1/reducer'
+import { walletConnectDisplayedInfoSelector } from 'src/walletConnect/v1/selectors'
 
 export const useIsDappListed = (dappName?: string) => {
   const dappsList = useSelector(dappsListSelector)
+  const walletConnectDisplayedInfo = useSelector(walletConnectDisplayedInfoSelector)
 
   return (
-    !!dappName && !!dappsList.find((dapp) => dapp.name.toLowerCase() === dappName.toLowerCase())
+    walletConnectDisplayedInfo === WalletConnectDisplayedInfo.None ||
```

Comment:
Seems a bit misleading/dangerous to link the listed status to the DIsplayedInfo remote config.
I think this should be put outside of this hook.

Reply:
yeah good point. i was only thinking that we want this for all enum values other than `none` but indeed that's a bit dangerous since we're using free text on firebase. i changed these conditions to check for `basic` instead, when/if we get to implementing the `full` variant we can tweak this code